src/dotnetTestExplorer.ts

```diff
@@ -198,7 +215,7 @@ export class DotnetTestExplorer implements TreeDataProvider<TestNode> {
         return new Promise((c, e) => {
             try {
                 const results = Executor
-                    .execSync(`dotnet test -t -v=q${this.checkBuildOption()}${this.checkRestoreOption()}`, this.testDirectoryPath)
+                    .execSync(`dotnet test -t -v=q${this.getDotNetTestOptions()}`, this.testDirectoryPath)

```

Comment:
You did not add `${this.outputTestResults()}`, so the test result will not be shown when VS Code starts. Is it by design?

Reply:
Oh, please ignore. It only lists the tests.